### DISCUSSION TITLE
Error while deploying on netlify

### DISCUSSION THREAD

**Discussion by kiranshetty07061999** [iTwin/itwinjs-core#2795] [Q&A]
Hey i tried deploying the itwin app on netlify... i got this error...any idea on how to resolve this

error_description=The+requested+scope+is+invalid%2C+unknown%2C+malformed%2C+or+exceeds+that+which+the+client+is+permitted+to+request.&state=f6284243660f4bfb9febb35425a159d5&error=invalid_scope#.

**Comment by aruniverse** [comment]
Did you follow this [medium article](https://medium.com/itwinjs/deploying-the-itwin-viewer-to-a-web-host-d45c5cfdf0cf)?

Are your env variables configured correctly? They need to match exactly how you have your client configured in the [developer portal](https://developer.bentley.com/my-apps/). Like the error says, theres a mismatch with the scopes (or redirects).

**Reply by kiranshetty07061999** [reply]
yes....I did follow the article...the env is configured properly...now it just says logging in...and doesnt really log in...

**Reply by aruniverse** [reply]
Which issuer are you using, `ims.bentley.com` or `imsoidc.bentley.com` ? Have you seen this other [post](https://medium.com/itwinjs/one-scope-to-call-them-all-aedc9840076b) related to auth? 

Could you tell us what version of @itwin/web-viewer-react and the itwnjs core packages you are using?

**Reply by kiranshetty07061999** [reply]
iam using ims.bentley.com and the version of the packages is 
  "@itwin/web-viewer-react": 1.3.0,
  itwinjs core =2.19.17,

**Reply by aruniverse** [reply]
Just to confirm, you're requesting the following scopes? `itwinjs organization profile email openid` 
Does the app work locally for you? If it works locally and not deployed, then its an issue with the redirects, which need to match **exactly** how they're configured in the developer portal. Make sure you have `https` as well as have the `netlify.toml` file to handle redirects.

**Comment by aruniverse** [comment]
Side note this isn't an issue with itwinjs, moving this to a discussion
